Tasks/SSH/ssh.ts

```diff
@@ -94,6 +95,18 @@ async function run() {
                 tl.debug('Copying script to remote machine.');
                 await sshHelper.copyScriptToRemoteMachine(scriptFile, scpConfig);
 
+                //change the line encodings
+                var isWin = os.type().match(/^Win/);
+                var windowsEncodedRemoteScriptPath = remoteScriptPath;
+                if (isWin) {
+                    remoteScriptPath = remoteScriptPath + ".unix";
+                    tl.debug('Fixing the line endings in case the file was created in windows');
+                    tl._writeLine('tr -d \'\\015\' <' + windowsEncodedRemoteScriptPath + ' > ' + remoteScriptPath);
+                    await sshHelper.runCommandOnRemoteMachine(
+                        'tr -d \'\\015\' < ' + windowsEncodedRemoteScriptPath + ' > ' + remoteScriptPath, 
+                        sshClientConnection, remoteCmdOptions);
+                }
```

Comment:
Instead of duplicating formation of the `tr...` string, could you save it in a variable and use it in both places?